### PR TITLE
Com292

### PR DESCRIPTION
**Pull request by Matt-Crow**
Had to pivot on client feedback, but admin configuration now works. @rramsdany , you might have to clean up a bit of my work on that page, but I imitated your style as best I could, so it doesn't look like the old booking page lol.

### CODE REVIEW

**Comment by Matt-Crow** [comment]
I just realized there appear to be a lot of files changed here, but that's because we haven't merged my other branch into main yet, so it should clear up quite a bit once we resolve that.

File: angular-app/src/app/app.module.ts [hunk 1]
```diff
@@ -170,7 +167,7 @@ import { AdminPanelWidgetComponent } from './theme/widgets/admin-panel-widget/ad
     DateManager,
     ScheduleService,
     UserService,
-    AppointmentService
```
Comment (rramsdany) [inline angular-app/src/app/app.module.ts:173]: Why is `AppointmentService` being removed from the declarations array? Perhaps this wasn't intentional?

Comment (Matt-Crow) [reply angular-app/src/app/app.module.ts:173]: It's already in the `providers` array on line 165, so I removed the second one. I think we had two different versions of `AppointmentService`, and a merge just left both of them there.

Comment (rramsdany) [reply angular-app/src/app/app.module.ts:173]: Ah gotcha, did not catch that it was already there.

**Review by rramsdany — APPROVED** [review]
Looks great, can't wait to try it out

**Review by Rojanmhj2055 — APPROVED** [review]

**Pull request merged**